packages/azure-http-specs/specs/azure/resource-manager/operation-templates/mockapi.ts

```diff
@@ -0,0 +1,62 @@
+import { json, passOnSuccess, ScenarioMockApi } from "@typespec/spec-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+const SUBSCRIPTION_ID_EXPECTED = "00000000-0000-0000-0000-000000000000";
+const validCheckNameAvailability = {
+  nameAvailable: true,
+  reason: "AlreadyExists",
+  message: "",
+};
```

Comment:
This is not same as your scenario doc

Reply:
Fixed by the [commit#d70464f](https://github.com/Azure/typespec-azure/pull/1934/commits/d70464f7251a63363cd3b47743600f95df5e1fd9)